lab_6_pipeline/pipeline.py

```diff
@@ -128,11 +218,18 @@ def convert_pos(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Extracts and converts POS from the OpenCorpora tags into the UD format
         """
+        return self._tag_mapping[self.pos][tags.POS or 'UNKN']
 
     def convert_morphological_tags(self, tags: OpencorporaTagProtocol) -> str:  # type: ignore
         """
         Converts the OpenCorpora tags into the UD format
         """
+        ud_tags = {}
+        for category in ( 'animacy', 'case', 'gender', 'number'):
+            if not (open_corpora_tag := eval(f'tags.{category}')):
```

Comment:
using eval is a bad practice, I would even ask to forget about it for 99.99% cases. rework

Reply:
fixed